exampleSite/content/docs/guide/shortcodes/tabs.ja.md

```
@@ -20,23 +20,23 @@ next: /docs/guide/deploy-site
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)